src/vstart.sh

```diff
@@ -542,6 +544,13 @@ start_mon() {
 			--cap mgr 'allow *' \
 			"$keyring_fn"
 
+		prun $SUDO "$CEPH_BIN/ceph-authtool" --gen-key --name=client.rgw --set-uid=0 \
+		    --cap mon 'allow *' \
+		    --cap osd 'allow *' \
+		    --cap mds 'allow *' \
+		    --cap mgr 'allow *' \
+		    "$keyring_fn"
```

Comment:
since we're not appending in ceph.conf anymore the keyring isn't strictly needed anymore,while there is an advantage that we could have rgw specific section under that config. the disadvantage is that we have to specify a `--name` when start rgw if I'm not mistaken... no strong opinions but we could do away with not creating a keyring I suppose? @cbodley thoughts?